### PR TITLE
Surface the text from the OSV elevator pitch

### PR DESCRIPTION
**Pull request by andrewpollock**
This can be referenced in the upcoming blog for USNs.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -6,9 +6,12 @@ nav_order: 1
 
 [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/google/osv.dev/badge)](https://api.securityscorecards.dev/projects/github.com/google/osv.dev)
 
-[osv.dev](https://osv.dev) is a [vulnerability database](https://osv.dev/list)
-and triage infrastructure for open source projects aimed at helping both open
-source maintainers and consumers of open source.
+OSV seeks to reduce the complexities of open-source software security
```
Comment (oliverchang) [inline docs/index.md:9]: nit: "open source" without the - for consistency with the rest of the repo?

Comment (oliverchang) [inline docs/index.md:9]: per my comment on the doc, I wonder if it's better for us to start with this instead:

"OSV enables developers to identify known 3P OSS vulnerabilities that pose genuine risk to their application and its environment, so they can focus remediation efforts on the vulnerabilities that matter and sustainably manage vulnerabilities that do not affect them."

And then briefly expand on the schema, OSV.dev infra (this repo), and OSV-Scanner ?

Comment (andrewpollock) [reply docs/index.md:9]: Adjusted in 7bbb953

File: docs/index.md [hunk 2]
```diff
@@ -22,6 +24,9 @@ through bisection and version analysis.
 Further information on the infrastructure architecture is available
 [here](contributing/architecture.md).
 
+[OSV-Scanner](https://google.github.io/osv-scanner/), is the first-party tool
```
Comment (oliverchang) [inline docs/index.md:27]: nit: no comma after OSV-Scanner?

**Review by oliverchang — APPROVED** [review]
LGTM with a nit

File: docs/index.md (hunk 2 above)
Comment (andrewpollock) [reply docs/index.md]: Fixed in https://github.com/google/osv.dev/pull/2112/commits/baa3c85781da3411e62d5beb9955558b34bd7202

**Pull request merged**